piccolo_admin/endpoints.py

```diff
@@ -411,7 +411,13 @@ class FormConfigResponseModel(BaseModel):
     description: t.Optional[str] = None
 
 
-def handle_auth_exception(request: Request, exc: Exception):
+@dataclass
+class SessionExpiryConfig:
+    session_expiry: timedelta
+    max_session_expiry: timedelta
+    increase_expiry: t.Optional[timedelta]
```

Comment:
```suggestion
@dataclass
class SessionExpiryConfig:
    session_expiry: timedelta
    max_session_expiry: timedelta
    increase_expiry: t.Optional[timedelta] = timedelta(minutes=20)
```

Adding a default argument here inline with prior defaults so if we do move to end users constructing this then they don't need to provide the argument if they don't want to.